components/hab/src/command/plan/init.rs

```diff
@@ -115,10 +117,40 @@ pub fn start(ui: &mut UI,
                your habitat. There are several hooks to create and tweak! See the full list \
                with info here: https://www.habitat.sh/docs/reference/plan-syntax/#hooks"));
 
+    try!(render_gitignore(ui, &root));
+
     try!(ui.end("A happy abode for your code has been initialized! Now it's time to explore!"));
     Ok(())
 }
 
+fn render_gitignore(ui: &mut UI, root: &str) -> Result<()> {
```

Comment:
We should probably only do this if we detect a `.git` directory somewhere in the file path above us. Otherwise poor bazaar users will get a file they don't want.